### ISSUE TITLE
Revise QueryBuilder notebook

### ISSUE THREAD

**Issue by yakutovicha** [aiidateam/aiida-tutorials#68]
```
---------------------------------------------------------------------------
ImportError                               Traceback (most recent call last)
<ipython-input-3-464cc73cea91> in <module>()
      7     load_dbenv()
      8 from aiida.orm import load_node, Node, Group, Computer, User
----> 9 from aiida.orm import CalculationFactory, DataFactory
     10 from aiida.orm.calculation.job import JobCalculation
     11 from aiida.orm.querybuilder import QueryBuilder

ImportError: cannot import name 'CalculationFactory'
```
should it be:
`from aiida.plugins.factories import CalculationFactory`
??

**Comment by yakutovicha** [comment]
```
---------------------------------------------------------------------------
ModuleNotFoundError                       Traceback (most recent call last)
<ipython-input-1-d024146f8d58> in <module>()
      8 from aiida.orm import load_node, Node, Group, Computer, User
      9 from aiida.plugins.factories import CalculationFactory, DataFactory
---> 10 from aiida.orm.calculation.job import JobCalculation
     11 from aiida.orm.querybuilder import QueryBuilder
     12 

ModuleNotFoundError: No module named 'aiida.orm.calculation'
```

**Comment by ltalirz** [comment]
@yakutovicha didn't we discuss you would be updating this?

**Comment by sphuber** [comment]
@yakutovicha I cannot find these imports anywhere. Where did you find them? Have they already been fixed in PR #75 ?

**Comment by yakutovicha** [comment]
@sphuber I was following the instructions [here](https://aiida-tutorials.readthedocs.io/en/tutorial_2019_05/pages/2019_MARVEL_Psik_MaX/sections/querybuilder.html). And according to them the example was in `aiida-demos/tutorial/` folder

**Comment by yakutovicha** [comment]
> @yakutovicha didn't we discuss you would be updating this?

sorry @ltalirz, I self-assigned

**Comment by sphuber** [comment]
Hang on @yakutovicha , I just finished the fix. I was just about to push

**Comment by sphuber** [comment]
Fixed in PR #80
